Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/Migration/Tools/MigrationWindow.cs

```diff
@@ -0,0 +1,417 @@
+                GUI.enabled = false;
+            }
+        }
+
+        private void DrawMigrationToolbars()
+        {
+            using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                EditorGUILayout.Space();
+
+                ToolbarOption previousSelectedToolbar = selectedToolbar;
+                selectedToolbar = (ToolbarOption)GUILayout.Toolbar((int)selectedToolbar, toolbarTitles);
+
+                if (previousSelectedToolbar != selectedToolbar)
+                {
+                    scrollPosition = Vector2.zero;
+                    migrationObjects.Clear();
+                }
+
+                if (selectedToolbar == ToolbarOption.Project)
+                {
+                    DrawProjectToolbar();
+                }
+                else
+                {
+                    DrawObjectsToolbar();
+                }
+            }
+        }
+
+        private void DrawObjectsToolbar()
+        {
+            EditorGUILayout.Space();
+
+            using (new GUILayout.HorizontalScope())
+            {
+                string tooltip = $"Drag and Drop {toolbarTitles[(int)selectedToolbar]} for Migration.";
+                EditorGUILayout.LabelField(new GUIContent($"{toolbarTitles[(int)selectedToolbar]} Selection", InspectorUIUtility.InfoIcon, tooltip));
+
+                var selectionType = selectedToolbar == ToolbarOption.Objects ? typeof(GameObject) : typeof(Object);
+                var allowSceneObjects = selectedToolbar == ToolbarOption.Objects;
+
+                selection = EditorGUILayout.ObjectField(null, selectionType, allowSceneObjects);
+                if (selection)
+                {
+                    AddSelection();
+                }
+            }
+            EditorGUILayout.Space();
+
+            using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                DrawMigrationObjectsList();
+            }
+        }
+
+        private void DrawProjectToolbar()
+        {
+            EditorGUILayout.Space();
+            using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
+            {
+                using (new GUILayout.VerticalScope(EditorStyles.helpBox))
+                {
+                    if (GUILayout.Button("Migrate"))
+                    {                        
+                        MigrateAllPrefabs();
+                        MigrateAllScenes();
+                    }
+
+                    using (new EditorGUILayout.HorizontalScope())
+                    {
+                        GUILayout.FlexibleSpace();
+
+                        string tooltip = "All Scenes and Prefabs Selected";
+                        EditorGUILayout.LabelField(new GUIContent(tooltip, InspectorUIUtility.WarningIcon));
+                    }
+                }
+            }
+        }
+
+        private void DrawMigrationObjectsList()
+        {
+            scrollPosition = EditorGUILayout.BeginScrollView(scrollPosition);
```

Comment:
suggestion: use ScrollViewScope() to mark content that is scrolling
https://docs.unity3d.com/ScriptReference/EditorGUILayout.ScrollViewScope-ctor.html

Reply:
Nice One

Reply:
Changed